advancedTodo/src/redux/todoAppComplete/todoApp.ui.js

```diff
@@ -1,32 +1,33 @@
 import React, { Component } from 'react';
 import { isUndefined, isEmpty } from 'lodash';
 import { NotificationContainer, NotificationManager } from 'react-notifications';

```

Comment:
You are now importing `DELETE_MESSAGE`, `SUCCESS_MESSAGE`, and `TODO_COMPLETED_MESSAGE` but not using them.  Consider removing the unused imports.